kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -307,6 +307,23 @@
           this.replacements.length === this.selectedActiveQuestions.length
         );
       },
+      replaceSelectedQuestionsString() {
+        const x = this.selectedActiveQuestions.length - this.replacements.length;
```

Comment:
I think we should have a more descriptive name than `x` here to help clarify the code.